stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -85,6 +85,7 @@ spec:
                 add:
                 - NET_BIND_SERVICE
             runAsUser: {{ .Values.controller.image.runAsUser }}
+            allowPrivilegeEscalation: true
```

Comment:
Instead of explicitly setting this value add it to the values file and pull it in here. This way people can set it true or false depending on their environment.

Reply:
Done :)